modules/programs/smug.nix

```diff
@@ -59,6 +59,14 @@ in
                           '';
                         };
 
+                        selected = lib.mkOption {
+                          type = lib.types.nullOr lib.types.bool;
+                          default = null;
+                          description = ''
+                            Whether this windows should be selected by default at session start.
```

Comment:
Shouldn't this be `this window` in stead of `this windows`?

Reply:
```suggestion
                            Whether this window should be selected by default at session start.
```